Manifest.toml

```diff
@@ -0,0 +1,199 @@
+# This file is machine-generated - editing it directly is not advised
```

Comment:
I don't check in Manifest.toml's normally, since they are version dependent and can contain paths to local files.

Reply:
Ah ok, I wasn't sure if it was needed or not.